src/Test/Perf/bootstrap.bat

```diff
@@ -0,0 +1,9 @@
+@echo off
+
+call ..\..\..\Restore.cmd
```

Comment:
Don't use relative paths.  Bootstrap should be callable from anywhere, not just from the same directory that bootstrap.bat is in.